vtr_flow/scripts/python_libs/vtr/parse_vtr_task.py

```diff
@@ -209,6 +209,9 @@ def parse_task(config, config_jobs, flow_metrics_basename=FIRST_PARSE_FILE, alt_
     max_circuit_len = len("circuit")
     for job in config_jobs:
         work_dir = job.work_dir(get_latest_run_dir(find_task_dir(config, alt_tasks_dir)))
+        job.parse_command()[0] = work_dir
```

Comment:
Not sure if a comment would be helpful here or this is obvious in context @saaramahmoudi